### DISCUSSION TITLE
Allow checking firmware update for list of devices and firmware versions

### DISCUSSION THREAD

**Discussion by pentamassiv** [fwupd/fwupd#4970] [Q&A]
**Is your feature request related to a problem? Please describe.**
I have multiple laptops (one small, light one for traveling for example) that I rarely turn on. When I do turn it on, it usually is because I need it and not to check for firmware updates. Another example would be my Raspberry Pi. It is always running but I rarely connect to it to update its software or check for firmware updates. Manually checking if an update exists of course is possible, but it would be nice if I could automatically check it from my main machine.

**Describe the solution you'd like**
It would be nice if the following was possible
- ssh into travel laptop
- fwupdmgr get-devices > travel_laptop_devices.txt
- sync travel_laptop_devices.txt to main computer
- fwupdmgr get-updates travel_laptop_devices.txt
- Then I would only need to turn on my travel laptop if there is a firmware update available

**Describe alternatives you've considered**
Manually turn on laptop every few weeks and check for firmware update on it

**Additional context**

**Comment by superm1** [comment]
So I think you can actually accomplish this with host emulation on main.  If you build an emulation JSON file and load it into a daemon on another machine you should be able to check for updates, no?
